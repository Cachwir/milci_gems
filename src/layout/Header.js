import React from "react";
import {Link} from "react-router-dom";

function Header()
{
    return (
        <div>
            <div id="top-line"></div>

            <div className="container">

                {/* Header */}
                <header id="header">
                    {/* Logo */}
                    <div className="ten columns">
                        <div id="logo">
                            <h1><Link to="/"><img src={process.env.PUBLIC_URL + "/assets/images/logo.png"} alt="logo" /></Link></h1>
                            <div id="tagline">Des gemmes d'exception</div>
                            <div className="clearfix"></div>
                        </div>
                    </div>

                    {/* Social / Contact */}
                    <div className="six columns">

                        <div className="clearfix"></div>

                    </div>
                </header>

                <div className="clearfix"></div>

            </div>
        </div>
    );
}

export default Header;