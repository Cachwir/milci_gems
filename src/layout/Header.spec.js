import { shallow } from 'enzyme'
import { expect } from 'chai'
import React from 'react'
import Header from './Header'

describe('<Header />', () => {
    it('should match its reference snapshot', () => {
        const wrapper = shallow(
            <Header />
        );

        expect(wrapper).to.matchSnapshot();
    });
});