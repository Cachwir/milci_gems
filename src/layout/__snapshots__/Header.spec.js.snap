// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should match its reference snapshot 1`] = `
<div>
  <div
    id="top-line"
  />
  <div
    className="container"
  >
    <header
      id="header"
    >
      <div
        className="ten columns"
      >
        <div
          id="logo"
        >
          <h1>
            <Link
              to="/"
            >
              <img
                alt="Nevia Premium Template"
                src="/assets/theme/images/logo.png"
              />
            </Link>
          </h1>
          <div
            id="tagline"
          >
            Template Without Compromises!
          </div>
          <div
            className="clearfix"
          />
        </div>
      </div>
      <div
        className="six columns"
      >
        <div
          className="contact-details"
        >
          Contact : cachwir@gmx.com
        </div>
        <div
          className="clearfix"
        />
      </div>
    </header>
    <div
      className="clearfix"
    />
  </div>
</div>
`;
